app/css/NameTag.css

```diff
@@ -1,47 +1,46 @@
 /* NameTag.css */
 .label-text {
   font-weight: bold;
-  font-size: 30px;
+  font-size: calc(1.875rem + 0.5vw); /* 30px equivalent, responsive to system settings and screen size */
```

Comment:
How was the ```1.875rem``` derived?

Reply:
I just converted px to rem based on root font size of 16 px.

Reply:
30 px = 1.875 rem

Reply:
Resolved